base/src/main/java/uk/ac/ebi/atlas/testutils/SolrUtils.java

```diff
@@ -0,0 +1,25 @@
+package uk.ac.ebi.atlas.testutils;
+
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.atlas.solr.cloud.SolrCloudCollectionProxyFactory;
+import uk.ac.ebi.atlas.solr.cloud.fullanalytics.AnalyticsCollectionProxy;
+import uk.ac.ebi.atlas.solr.cloud.search.SolrQueryBuilder;
+
+import javax.inject.Inject;
+
+import static uk.ac.ebi.atlas.solr.cloud.fullanalytics.AnalyticsCollectionProxy.BIOENTITY_IDENTIFIER;
+
+@Component
+public class SolrUtils {
+
+    @Inject
+    private SolrCloudCollectionProxyFactory solrCloudCollectionProxyFactory;
+
+    public String fetchRandomExpressionAtlasGene() {
+        AnalyticsCollectionProxy analyticsCollectionProxy = solrCloudCollectionProxyFactory.createAnalyticsCollectionProxy();
+        SolrQueryBuilder<AnalyticsCollectionProxy> queryBuilder = new SolrQueryBuilder<>();
+        queryBuilder.setFieldList(BIOENTITY_IDENTIFIER);
+        queryBuilder.setRows(10000);
+        return analyticsCollectionProxy.query(queryBuilder).getResults().get(1000).getFieldValue("bioentity_identifier").toString();
```

Comment:
This is still not very random 😃 It will always return the gene with index 1000.

Reply:
The proper way to do this would be to have a dynamic field with the name `random_*` that Solr interprets as a random source of data, so you can sort by `random_<random integer>`, but we don’t have such dynamic fields and I wouldn’t bother for testing given that it’s enough with the suggestion above.

Reply:
Also, rename the method to something like `fetchRandomGeneFromAnalytics` to know the source of the gene.